test/pubsub-disruption/mzcompose.py

```diff
@@ -58,10 +58,11 @@ class Disruption:
         breakage=lambda c: c.kill("toxiproxy"),
         fixage=lambda c: toxiproxy_start(c),
     ),
+    # docker compose pause has become unreliable recently
     Disruption(
```

Comment:
Do we have further usages of this pattern (besides https://github.com/MaterializeInc/materialize/pull/23559/files)?
If so, does it make sense to extract this logic / definition? (optional)

Reply:
Only these two.